<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Eduardo Volpi de Almeida</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>991</width>
    <height>586</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>991</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C++ programming</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/imagens/c++.ico</normaloff>:/imagens/c++.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>82</y>
      <width>951</width>
      <height>451</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabExemplos1">
     <attribute name="title">
      <string>Exemplos 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Comandos e Loops</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>24</y>
         <width>161</width>
         <height>150</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnIfElse">
          <property name="statusTip">
           <string>Exemplo de comando if else</string>
          </property>
          <property name="text">
           <string>If else</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnForLoop">
          <property name="text">
           <string>for loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWhileLoop">
          <property name="text">
           <string>while loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDoWhileLoop">
          <property name="text">
           <string>do while loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>switch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>181</width>
        <height>330</height>
       </rect>
      </property>
      <property name="title">
       <string>MessageBox</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>161</width>
         <height>206</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QPushButton" name="btnSobre">
          <property name="text">
           <string>Sobre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSobreQt">
          <property name="text">
           <string>Sobre Qt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInfo">
          <property name="text">
           <string>Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnErro">
          <property name="text">
           <string>Erro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAlerta">
          <property name="text">
           <string>Alerta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuestao">
          <property name="text">
           <string>Questão</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuestao2">
          <property name="text">
           <string>Questão 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>40</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/imagens/messagebox.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>181</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Estilos da Aplicação</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>30</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/imagens/styles.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>172</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="btnDefinirEstilo">
          <property name="text">
           <string>Definir Estilo</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>87</y>
         <width>76</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QRadioButton" name="rbFusion">
          <property name="text">
           <string>Fusion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbPadrao">
          <property name="text">
           <string>Padrão</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWindows">
          <property name="text">
           <string>Windows</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>375</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#3fce51;&quot;&gt;Input&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#747474;&quot;&gt;Box&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>11</y>
        <width>181</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>InputBox</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>161</width>
         <height>170</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(91, 91, 91);
color: rgb(255, 255, 255);
padding-top:0px;
border-radius:3;</string>
          </property>
          <property name="text">
           <string>com tradução do Qt</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputTexto2">
          <property name="text">
           <string>Texto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputSenha2">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputInt2">
          <property name="text">
           <string>Número Int</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputDouble2">
          <property name="text">
           <string>Número Double</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputItem2">
          <property name="text">
           <string>Seleção de Item</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(91, 91, 91);
color: rgb(255, 255, 255);
padding-top:0px;
border-radius:3;</string>
          </property>
          <property name="text">
           <string>com tradução no código</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputTexto">
          <property name="text">
           <string>Texto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputSenha">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputInt">
          <property name="text">
           <string>Número Int</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputDouble">
          <property name="text">
           <string>Número Double</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnInputItem">
          <property name="text">
           <string>Seleção de Item</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>213</x>
        <y>346</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#747474;&quot;&gt;Message&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic; color:#3fce51;&quot;&gt;Box&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>40</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">padding:10;</string>
      </property>
      <property name="text">
       <string>Data/Hora</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/imagens/calendar_80px.png</normaloff>:/imagens/calendar_80px.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="btnPastasDoSistema">
         <property name="text">
          <string>Pastas do Sistema</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnParaTestes">
         <property name="text">
          <string>para testes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnAbrirTelaDois">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>180</y>
        <width>158</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>  Abrir Tela Dois</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/imagens/application_double.png</normaloff>:/imagens/application_double.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>310</y>
        <width>78</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabExemplos2">
     <attribute name="title">
      <string>Exemplos 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>14</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/imagens/qt_100px2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>17</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/imagens/c++_50px.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblTestes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>38</y>
      <width>581</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>std::cout &lt;&lt; &quot;C++, unmatched power, control, and performance.&quot; &lt;&lt; std::endl;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>23</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-style:italic; color:#747474;&quot;&gt;Code Examples&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="action_Novo"/>
    <addaction name="actionAbrirTelaDois"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menu_Arquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Novo">
   <property name="text">
    <string>&amp;Novo</string>
   </property>
  </action>
  <action name="actionAbrirTelaDois">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::Computer"/>
   </property>
   <property name="text">
    <string>Abrir Tela Dois</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/imagens/inputbox.png</normaloff>:/imagens/inputbox.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Sair</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
